environments/README.md

```diff
@@ -0,0 +1,21 @@
+# Environments
+
+This directory contains the environment files for the various environments needs to run the analyses in this repository.
+
+
+## Environment files
+- `Cell_profiling_env.yml`
+- `Cell_profiling_processing_env.yml`
+- `R_env.yml`
+- `SAM_env.yml`
```

Comment:
Consider going more in-depth here with the documentation and making it clear where these different environments are needed (e.g., at the module/folder level or you can even go to the notebook level). It will make it easier for reproducibility in my opinion.

Reply:
Good call. Added more documentation!